articles/deep-neural-system-matlab/index.md

```diff
@@ -0,0 +1,214 @@
+### How to create a deep neural network using MATLAB.
+### Introduction.
+Neural networks form the base of deep learning and a subfield of machine learning where algorithms are inspired by the structure of the human brain. It takes in data train themselves to recognize the pattern in this data and then predict the output for a new set of similar data.
+MATLAB provides a deep learning toolbox for implementing the deep neural network. MATLAB includes the digit dataset so you don't need to download it from an external source.
+In this article, we will look at how to prepare your digit dataset, the structure of a convolution neural network. We will also look at the Matlab code for training and validation and code for discrete testing.
+
+### Prerequisite.
+1. Knowledge in Matlab.
+2. Matlab installed on your computer.
```

Comment:
Consider adding links to Matlab download site. Reference a getting started article that is already published [here](https://www.section.io/engineering-education/getting-started-with-matlab/).